server/JsonStreamingServer/JsonStreamingServer.WebApi/JsonStreamingServer.WebApi.csproj

```diff
@@ -8,12 +8,14 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="7.0.3" />
+    <PackageReference Include="ServiceStack.Text" Version="6.6.0" />
```

Comment:
No need to add another dependency to the Web API project

Reply:
Removed